code/modules/maps/overmap/space/alienship.dm

```diff
@@ -116,62 +116,62 @@
 	. = ..()
 	dump_area = locate(/area/tether_away/alienship/equip_dump)
 
-/area/shuttle/excursion/away_alienship/shuttle_arrived()
```

Comment:
all of this were actually removed in overmaps update years ago but is still sitting around.

Reply:
I was curious about this, seen it a while ago. Makes sense it's just legacy stuff